source/_docs/environment-specific-config.md

```diff
@@ -22,7 +47,7 @@ Filters or functions running in an mu-plugin can enable development plugins that
 
 ### Create the Plugin
 
-Copy this plugin file to `wp-content/mu-plugins/mu-plugins/site-config.php` and edit accordingly.
+Copy this plugin file to `wp-content/mu-plugins/site-config.php` and edit accordingly.
 
```

Comment:
Not a blocker, just would like to understand this change -- is this a preference or are there broader implications of mu vs standard plugin?

Reply:
@rachelwhitton the issue here is that it was doubled, easy to miss.